templates/peering-test.yaml

```diff
@@ -0,0 +1,236 @@
+apiVersion: cluster.x-k8s.io/v1alpha4
```

Comment:
let's remove this test template from templates (these get published in the release)